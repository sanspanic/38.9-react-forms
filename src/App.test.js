import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("ok: renders without crashing", function () {
  render(<App />);
});

test("ok: matches snapshot", function () {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
