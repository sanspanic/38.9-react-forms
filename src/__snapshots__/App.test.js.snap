// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ok: matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div />
    <form>
      <label
        for="color"
      >
        Color: 
      </label>
      <input
        id="color"
        name="color"
        placeholder="color"
        type="text"
        value=""
      />
      <label
        for="width"
      >
        Width: 
      </label>
      <input
        id="width"
        name="width"
        placeholder="width"
        type="text"
        value=""
      />
      <label
        for="height"
      >
        Height: 
      </label>
      <input
        id="height"
        name="height"
        placeholder="height"
        type="text"
        value=""
      />
      <button>
        Add new
      </button>
    </form>
  </div>
</DocumentFragment>
`;
